tracer/samples/AzureFunctionsWithAgentLessLogging/FunctionWithLogging/run.ps1

```diff
@@ -0,0 +1,133 @@
+<#
+.SYNOPSIS
+    A script that allows the user to run C# sample applications to demonstrate
+    the setup and usage of Datadog's .NET Tracer log collection and log trace ID injection in a function
+.PARAMETER AppDirectory
+    The application directory - defaults to the location of this script and need not be set
+    by the user unless samples are moved around.
+.PARAMETER ApiKey
+    The Datadog API key (only required when -Agentless is set)
+.EXAMPLE
+    pwsh BuildAndRunSample.ps1 -ApiKey YOUR_API_KEY_HERE
+#>
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $false)]
+    [System.String]
+    $AppDirectory, # only necessary if the script is removed from its current directory layout
+
+    [Parameter(Mandatory = $false)]
+    [System.String]
+    $ApiKey
+)
+
+<#
+.DESCRIPTION
+    Checks to ensure that the dotnet command line too is accessible.
+#>
+function Test-DotNetCLI {
+    try {
+        dotnet --version
+        return $true
+    }
+    catch {
+        return $false # dotnet CLI not installed
+    }
+}
+
+function Set-EnvironmentVariableForProcess {
+    param([string]$name, [string]$value)
+    Write-Host "Setting $name to $value"
+    [System.Environment]::SetEnvironmentVariable($name, $value, [System.EnvironmentVariableTarget]::Process)
+}
+
+<#
+.DESCRIPTION
+    Builds and runs the desired sample project.
+#>
+function BuildAndRunSample {
+    func start
+}
+
+
+if (-not (Test-DotNetCLI)) {
+    Write-Error "The dotnet CLI tool wasn't found - please ensure it is installed."
+    exit 1
+}
+
+## Configure logs injection by setting up Unified Service Tagging
+## cf https://docs.datadoghq.com/tracing/other_telemetry/connect_logs_and_traces/dotnet/?tab=serilog
+Set-EnvironmentVariableForProcess -name "DD_LOGS_INJECTION" -value "true"
+Set-EnvironmentVariableForProcess -name "DD_ENV" -value "dev"
+Set-EnvironmentVariableForProcess -name "DD_SERVICE" -value "LogsInjectionInFunctionSamples"
+Set-EnvironmentVariableForProcess -name "DD_VERSION" -value "1.0.0"
+Set-EnvironmentVariableForProcess -name "DD_API_KEY" -value $ApiKey
+Set-EnvironmentVariableForProcess -name "DD_LOGS_DIRECT_SUBMISSION_INTEGRATIONS" -value "ILogger"
+
+if ($IsWindows) {
+    Write-Host "Using Windows configuration"
+    $AppDirectory = Get-ApplicationDirectory
```

Comment:
I don't see `Get-ApplicationDirectory` defined anywhere. What if we change it to this?

```suggestion
    if ($null -ne $AppDirectory)
    {
        $AppDirectory = "$pwd"
    }
```